### PR TITLE
139 log reg

### PR DESCRIPTION
**Pull request by MaksGMaks**
In dialogs I make fields public to create access for DialogManager, which after successfull registration add text from registration dialog into login dialog. Code was tested in different input

### CODE REVIEW

File: _client/src/Ui/Dialogs/DialogManager.cpp [hunk 1]
```diff
@@ -99,6 +101,20 @@ void DialogManager::showMessage(const QString &title, const QString &message, QM
...
+void DialogManager::onLoginSuccess()
+{
+    m_loginDialog->hide();
+    showMessage(tr("Message"), tr("Login successfull"), QMessageBox::Information);
+}
+
+void DialogManager::onRegistrationSuccess()
+{
+    m_registrationDialog->hide();
+    m_loginDialog->m_usernameField->setText(m_registrationDialog->m_usernameField->text());
```
Comment (Sage-Cat) [inline _client/src/Ui/Dialogs/DialogManager.cpp]: Do not break encapsulation. 
Also, consider using already existing method `LoginDialog::showWithPresetData`

File: _client/src/Ui/Dialogs/LoginDialog.hpp [hunk 2]
```diff
@@ -35,9 +35,11 @@ protected slots:
     void onLoginClicked();
     void onRegisterLinkClicked();
 
-private:
+public:
```
Comment (Sage-Cat) [inline _client/src/Ui/Dialogs/LoginDialog.hpp]: encapsulation  is broken

File: _client/src/Ui/Dialogs/DialogManager.hpp [hunk 3]
```diff
@@ -40,6 +40,10 @@ class DialogManager : public QObject
 private:
     ApiManager &m_apiManager;
 
+public slots:
```
Comment (Sage-Cat) [inline _client/src/Ui/Dialogs/DialogManager.hpp]: protected or private slots

**Review by Sage-Cat — CHANGES_REQUESTED** [review]

File: _client/src/Ui/Dialogs/DialogManager.hpp (hunk 3 above)
Comment (MaksGMaks) [reply _client/src/Ui/Dialogs/DialogManager.hpp]: done

File: _client/src/Ui/Dialogs/LoginDialog.hpp (hunk 2 above)
Comment (MaksGMaks) [reply _client/src/Ui/Dialogs/LoginDialog.hpp]: done

**Review by Field-Effect-Transistor — APPROVED** [review]

**Review by Sage-Cat — APPROVED** [review]

**Pull request merged**